obp-commons/src/main/scala/com/openbankproject/commons/util/ReflectUtils.scala

```diff
@@ -24,7 +24,7 @@ object ReflectUtils {
     * @param includeVar whether include var values
     * @return map of val or var name to value
     */
-  def getNameToValues(obj: AnyRef, excludes: Seq[String] = Nil, includeVar: Boolean = true) = {
+  def getNameToValues(obj: AnyRef, excludes: Seq[String] = Nil, includeVar: Boolean = true): Map[String, Any] = {
```

Comment:
this file did not modify feature, only add explicit return type to every function, make them good readable.